### ISSUE TITLE
Update fabric-gateway module to fabric-rfcs

### ISSUE THREAD

**Issue by dexhunter** [hyperledger/fabric-sdk-py#153]
[Fabric-rfcs](https://hyperledger.github.io/fabric-rfcs/text/0000-fabric-gateway.html) describes the Fabric Gateway Feature and we can build based on [the source code](https://github.com/hyperledger/fabric-gateway)

**Comment by dexhunter** [comment]
cc @KivenChen

**Comment by jeriox** [comment]
Support for gateway would be much appreciated :)

**Comment by dexhunter** [comment]
@jeriox yes, I am wondering to write a project proposal for adding python support for fabric-gateway, are you interested in co-supervising/mentoring the project?

**Comment by jeriox** [comment]
@dexhunter unfortunately not, I just need it for my thesis and don't have the capacity to help out here

**Comment by dexhunter** [comment]
@jeriox No problem. I think I will first update python sdk to support fabric 2.4 and then check with others to see if we can add python support for fabric-proto. By the way, just wondering what's your use case for the python sdk?

**Comment by jeriox** [comment]
> @jeriox No problem. I think I will first update python sdk to support fabric 2.4 and then check with others to see if we can add python support for fabric-proto.

were you able to make any progress so far?

> By the way, just wondering what's your use case for the python sdk?

I have a lot of existing code dealing with the data that I store in hyperledger fabric that is only available in python. I wanted to save myself the effort of migrating the code to another language or passing the processed data around before actually storing it
